abi_wrapper.go

```diff
@@ -0,0 +1,37 @@
+	}
+
+	if val, ok := BundledContracts[contractFile]; ok {
+		return strings.NewReader(val)
+	}
+	if _, err := os.Stat(contractFile); os.IsNotExist(err) {
+		log.Fatalf("Cannot find the abi file: %v", err)
+	}
+	jsonReader, err := os.Open(contractFile)
+	if err != nil {
+		log.Fatalf("Cannot read the abi file: %v", err)
+	}
+	return jsonReader
+}
+
+func GetAbi(contractFile string) *abi.ABI {
```

Comment:
It feels little weird for this to handle both files and bundled named contracts. I get that the cmd does that for convenience, but this also forces users of the package to do the same. What about two different functions instead? Something like `ABIOpenFile()` and `ABIBuiltIn()`?